sklearn/model_selection/_validation.py

```diff
@@ -1594,7 +1603,22 @@ def permutation_test_score(
     fit_params : dict, default=None
         Parameters to pass to the fit method of the estimator.
 
-        .. versionadded:: 0.24
+        .. deprecated:: 1.6
+            This parameter is deprecated and will be removed in version 1.6. Use
+            ``params`` instead.
+
+    params : dict, default=None
+        Parameters to pass to the `fit` method of the estimator and to the scorer.
+
+            - If `enable_metadata_routing=False` (default):
+              Parameters directly passed to the `fit` method of the estimator.
+
+            - If `enable_metadata_routing=True`:
+              Parameters safely routed to the `fit` method of the estimator.
```

Comment:
and splitter, and scorer

Reply:
Done [e7acf1f](https://github.com/scikit-learn/scikit-learn/pull/29266/commits/e7acf1f9b07fd3e76d809a75856152fe73693d9e)